extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -177,6 +177,10 @@
   margin-bottom: 12px !important;
 }
 
+.i-amphtml-story-inline-page-attachment-chip-dark-theme {
+  background-color: rgba(0, 0, 0, 0.5) !important;
+}
+
```

Comment:
We can style these based on the `theme="dark"` on the parent.

Anywhere you're adding a new class we can just prepend that selector to scope it, like:
`[theme="dark"] .i-amphtml-story-inline-page-attachment-chip`

In that way we can remove these new classes and all the JS that adds them :)

Reply:
We can also take this one step further and use color variables that are set by theme using this pattern:
```
.i-amphtml-story-page-open-attachment {
  --ten-percent: rgba(255, 255, 255, 0.5);
}

[theme="dark"].i-amphtml-story-page-open-attachment {
  --ten-percent: rgba(0, 0, 0, 0.5);
}

.i-amphtml-story-inline-page-attachment-chip {
  background-color: var(--ten-percent)!important;
  ...
```

This wont work everywhere, like the inline SVG will need to be targeted with separately.
This is a good practice with color in general because it's scalable.